### PR TITLE
Add Kyro Network

### PR DESCRIPTION
**Pull request by ashtton**

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -1124,5 +1124,12 @@
         "addresses": [
             "vfnets.com"
         ]
-    }
+    },
+    {
+        "id": "kyro",
+        "name": "Kyro.gg",
```
Comment (GrowlyX) [inline servers.json:1130]: Please make this something other than the actual domain

**Review by GrowlyX — CHANGES_REQUESTED** [review]

**Comment by GleemingKnight** [comment]
Fixed.

**Review by GrowlyX — APPROVED** [review]

**Comment by GleemingKnight** [comment]
@GrowlyX What's up with this PR?

**Comment by GrowlyX** [comment]
> @GrowlyX What's up with this PR?

image

**Pull request closed without merging**